extensions/wikia/FacebookClient/scripts/preferences.js

```diff
@@ -54,7 +58,10 @@
 				callback: function (data) {
 					if (data.status === 'ok') {
 
-						window.GlobalNotification.show($.msg('fbconnect-preferences-connected'), 'confirm');
+						bannerNotification
+							.setContent($.msg('fbconnect-preferences-connected'))
```

Comment:
I know this code was already there, but we could introduce escaping messages: http://www.mediawiki.org/wiki/Manual:Messages_API#Use_of_the_messages